roles/system/defaults/main.yml

```diff
@@ -0,0 +1,119 @@
+---
+
+## Image
+image_name: "quay.io/3scale/porta"
+image_tag: "nightly"
+image_pull_policy: "Always"
+
+## GrafanaDashboard
+grafana_dashboard_label_key: "monitoring-key"
+grafana_dashboard_label_value: "middleware"
+
+## Environment
+env_amp_release: 2.7.1
+env_force_ssl: true
+env_ssl_cert_dir: "/etc/pki/tls/certs"
+env_threescale_sandbox_proxy_openssl_verify_mode: "VERIFY_NONE"
+env_threescale_superdomain: "localhost"
+env_threescale_provider_plan: "enterprise"
+env_rails_env: "preview"
+env_rails_log_level: "info"
+env_rails_log_to_stdout: true
+
+##
+## app
+##
+
+app_replicas: 2
+### app > probe > liveness
+app_liveness_probe_initial_delay_seconds: 30
+app_liveness_probe_timeout_seconds: 1
+app_liveness_probe_period_seconds: 10
+app_liveness_probe_success_threshold: 1
+app_liveness_probe_failure_threshold: 3
+### app > probe > readiness
+app_readiness_probe_initial_delay_seconds: 30
+app_readiness_probe_timeout_seconds: 5
+app_readiness_probe_period_seconds: 10
+app_readiness_probe_success_threshold: 1
+app_readiness_probe_failure_threshold: 3
+### app > resources
+app_resources_requests_cpu: "200m"
+app_resources_requests_memory: "1Gi"
+app_resources_limits_cpu: "400m"
+app_resources_limits_memory: "2Gi"
+### app > pdb
+app_pdb_enabled: true
+app_pdb_min_unavailable: 1
```

Comment:
Or max or min, both are not compatible at the same time, we have set on all workloads `max_unavail` as default
```suggestion
```